packages/ra-core/src/form/formMiddleware.ts

```diff
@@ -1,45 +0,0 @@
-import { LOCATION_CHANGE } from 'connected-react-router';
-import { destroy } from 'redux-form';
-import isEqual from 'lodash/isEqual';
-
-import { resetForm } from '../actions/formActions';
-import { REDUX_FORM_NAME } from './constants';
-
-/**
- * This middleware ensure that whenever a location change happen, we get the
```

Comment:
Do we have tests that prove that this still works?

Reply:
Yes, there are e2e tests for it